assembl/static2/tests/unit/components/debate/brightMirror/instructionView.spec.jsx

```diff
@@ -1,17 +1,121 @@
 import React from 'react';
-import ReactTestRenderer from 'react-test-renderer';
+import ShallowRenderer from 'react-test-renderer/shallow';
+import { connectedUserCan } from '../../../../../js/app/utils/permissions';
 
 import InstructionView from '../../../../../js/app/components/debate/brightMirror/InstructionView';
 
+jest.mock('../../../../../js/app/utils/permissions');
+
 describe('Bright Mirror page', () => {
   it('should match BrightMirror instruction snapshot', () => {
+    connectedUserCan.mockImplementation(() => true);
+
+    const timeline = [
+      {
+        identifier: 'brightMirror',
+        start: '1900-01-01T02:00:00Z',
+        end: '2200-01-01T00:00:00Z',
+        title: { entries: [{ en: 'brightMirror' }] }
+      }
+    ];
     const props = {
       announcementContent: {
         title: 'Instruction',
         body: 'Participez à la création de fiction'
+      },
+      isUserConnected: true,
+      ideaId: 0,
+      refetchIdea: Function,
+      posts: [],
+      timeline: timeline,
+      identifier: 'brightMirror'
+    };
+    const shallowRenderer = new ShallowRenderer();
+    shallowRenderer.render(<InstructionView {...props} />);
+    const result = shallowRenderer.getRenderOutput();
+    expect(result).toMatchSnapshot();
+  });
+
+  it('timeline is over', () => {
```

Comment:
what happens when timeline is over ? I would write something like "should tell the user if the timeline is over"